src/main/java/com/reality/relations/datavisual/DataInsertService.java

```diff
@@ -0,0 +1,4 @@
+package com.reality.relations.datavisual;
+
+public class DataInsertService {
```

Comment:
Create an interface & implement it. Because it will force the person who implements this interface to create the impl for necessary methods.